pkg/heroku/heroku.go

```diff
@@ -96,7 +116,7 @@ func (c *Client) Delete(path string) error {
 //   nil         no body
 //   io.Reader   body is sent verbatim
 //   else        body is encoded as application/json
-func (c *Client) NewRequest(method, path string, body interface{}) (*http.Request, error) {
+func (c *Client) NewRequest(method, path string, body interface{}, headers http.Header) (*http.Request, error) {
```

Comment:
i was trying to keep it similar to how the heroku client allows global additional headers:

https://github.com/remind101/empire/blob/master/pkg/heroku/heroku.go#L147

I also think your suggestion would work if everything that needed the request header was called from `APIReq`, but `run` accesses `NewRequest` directly:
https://github.com/remind101/empire/blob/master/cmd/emp/run.go#L125